packages/typespec-azure-vscode/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "typespec-azure-vscode",
+  "displayName": "typespec azure",
+  "description": "the extension for azure typespec",
+  "publisher": "typespec",
+  "version": "0.1.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Azure/typespec-azure.git"
+  },
+  "engines": {
+    "vscode": "^1.100.0"
+  },
+  "extensionDependencies": [
+    "typespec.typespec-vscode"
+  ],
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "onLanguage:typespec",
+    "workspaceContains:**/tspconfig.yaml"
+  ],
+  "main": "./dist/src/extension.cjs",
```

Comment:
as this extension is starting from scratch I would look into loading ESM modules directly instead as this is supposedly supported

add `type:  module` and update esbuild to produce esm

Reply:
this doesn't seem to have been resolved. The extension would just be `.js` otherwise node loads that as a common js module